integrations/coinbase/src/main/java/org/dash/wallet/integration/coinbase_integration/viewmodels/EnterAmountToTransferViewModel.kt

```diff
@@ -195,16 +195,18 @@ class EnterAmountToTransferViewModel @Inject constructor(
     private val maxAmountCoinbaseAccount: String
         get() {
             return coinbaseExchangeRate?.let {
-                it.coinBaseUserAccountData.balance?.amount ?: VALUE_ZERO
-            } ?: VALUE_ZERO
+                it.coinBaseUserAccountData.balance?.amount
+            } ?: CoinbaseConstants.VALUE_ZERO
         }
 
     private fun applyCoinbaseExchangeRate(amount: String): String {
         return coinbaseExchangeRate?.let { uiModel ->
-            val cleanedValue =
-                amount.toBigDecimal() / uiModel.currencyToDashExchangeRate.toBigDecimal()
+            val cleanedValue = amount
+                .replace(',', '.') // TODO: the amount sometimes comes here with a comma as decimal separator.
+                // TODO: it's better to identify the root of this and replace in there to prevent this problem from appearing anywhere else.
```

Comment:
It's not clear to me where this value is coming from, but it sometimes comes with a comma as a decimal separator.
Eventually, we should fix this at the point of receiving such value.